### PR TITLE
ci: PLT-353: add graceful shutdown script for celery

### PR DESCRIPTION
**Pull request by farioas**

### CODE REVIEW

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/bf95df2b304147000d3f4c1a85bfffcd8408b25d is created.

File: server/tasks/queue_terminator.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import time
+import argparse
+import logging
+import sys
+from celery.app.control import Control
+from process_file import app as celery
```
Comment (pakelley) [inline server/tasks/queue_terminator.py:6]: mind adding a quick/simple docstring explaining why we need this?

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/3d7df7f49c6bc84603ae927d3f7114474ec0cbed is created.

**Review by hakan458 — APPROVED** [review]

**Review by pakelley — APPROVED** [review]

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/64815946414a7cd65938a24c8b3fbcb09bc2c644 is created.

**Pull request merged**